src/main/java/net/smartcosmos/extension/tenant/rest/resource/tenant/CreateTenantResource.java

```diff
@@ -47,10 +43,9 @@
                     consumes = APPLICATION_JSON_UTF8_VALUE)
     @ConditionalOnProperty(prefix = ENDPOINT_ENABLEMENT_TENANTS_CREATE, name = ENDPOINT_ENABLEMENT_PROPERTY_ENABLED, matchIfMissing = true)
     public DeferredResult<ResponseEntity> createTenant(
-        @RequestBody @Valid RestCreateTenantRequest restCreateTenantRequest,
-        @AuthenticationPrincipal SmartCosmosUser user) {
+        @RequestBody @Valid RestCreateTenantRequest restCreateTenantRequest) {
 
-        return service.create(restCreateTenantRequest, user);
+        return service.create(restCreateTenantRequest, null);
```

Comment:
Technically, it's possible to create a new tenant with an authenticated user. Do we really want to lose the information whether a tenant was created by an already existing user or by an anonymous user?!